ironfish/src/wallet/database/transactionValue.test.ts

```diff
@@ -1,15 +1,25 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Transaction } from '../../primitives'
+import { createNodeTest, useAccountFixture } from '../../testUtilities'
 import { TransactionValue, TransactionValueEncoding } from './transactionValue'
 
 describe('TransactionValueEncoding', () => {
+  const nodeTest = createNodeTest()
+  let transaction: Transaction
+
+  beforeEach(async () => {
+    const account = await useAccountFixture(nodeTest.accounts)
+    transaction = await nodeTest.strategy.createMinersFee(BigInt(0), 1, account.spendingKey)
```

Comment:
It's better to move these to each test that uses it, and use a useMinersTxFixture()